manifests/t/th-ch/YouTubeMusic/1.18.0/th-ch.YouTubeMusic.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: th-ch.YouTubeMusic
+PackageVersion: 1.18.0
+MinimumOSVersion: 10.0.0.0

```

```suggestion
MinimumOSVersion: 10.0.0.0
Commands:
- youtube-music
```
to launch this app, you can type `youtube-music` in CMD or PowerShell.